renovate.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": ["github>redhat-developer/app-services-renovate-config"],
+  "dependencyDashboardApproval": true,
+  "separateMajorMinor": true,
+  "separateMultipleMajor": true,
```

Comment:
+1 on these two. I see `separateMajorMinor` should be `true` by default but perfectly fine to make it explicit